crates/line_point/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+#[macro_use]
+extern crate helix;
+
+ruby! {
+    class Line {
+        struct {
+            x1: f64,
+            y1: f64,
+            x2: f64,
+            y2: f64
+        }
+
+        def initialize(helix, p1: &Point, p2: &Point) {
+            Line { helix, x1: p1.x, y1: p1.y, x2: p2.x, y2: p2.y }
```

Comment:
There are currently some complexities around ownership. This is something that _can_ be done, but we need to do some work on our end for it.